contracts/WildcardSteward_v1.sol

```diff
@@ -0,0 +1,367 @@
+pragma solidity ^0.5.0;
+import "./ERC721Patronage_v1.sol";
+import "./ERC20PatronageReceipt_v1.sol";
+import "./interfaces/IERC20Mintable.sol";
+
+contract WildcardSteward_v1 is Initializable {
```

Comment:
As discussed we still have a bunch of things to do on this zuck :) 

Some points below think we still need to cover

2-Minting loyalty tokens (TBD whether we want this rate to be consistent or vary between tokens?) Done at set rate for now. 
3-Transfer token logic (reject a transfer if the new owner doesn't hold deposit, otherwise will auto foreclose) = see you have disabled transfers for now
4-Implement our business model
5-Adding certain events that are missing (emit when a token is bought etc etc)
6- Adding logic that allows the previous owner to buy back the token at a fixed price (+10%?) within a day or something (further more, we must build logic that then keeps the streak intact). Maybe 2 days for the owner to react is more reasonable?

Reply:
#2. My favourite option at the moment is to do the inverse of the harberger tax rate...
#3. Yes, I think ultimately we need to have a 2 step process for transfers (an approve). But for now disabling is the safe option.
#4. Yes, important. Putting it in code will clear things up!
#5. I think this is already mostly done. Maybe we will have a new event due to the implementation of the business model.
#6. Yeah, I think this is great. If we get notifications working really reliably then yeah. But to avoid sad complaints lets do 2 days greed!